models/intel/smartlab-object-detection-0004/README.md

```diff
@@ -0,0 +1,61 @@
+# smartlab-object-detection-0004
+
+## Use Case and High-Level Description
+
+This is a smartlab object detector that is based on YoloX for 416x416 resolution.
+
+## Example
+
+![](./assets/frame0001.jpg)
+
+## Specification
+
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| AP @ [ IoU=0.50:0.95 ]          | see PASCAL per-cls AP (internal test set) |
+| GFlops                          | 1.05                                      |
+| MParams                         | 0.9                                       |
+| Source framework                | PyTorch\*                                 |
+
+PASCAL per-cls AP:
+
+|    Class     |       per-cls AP         |
+|--------------|--------------------------|
+|  "weights",  |    0.41                  |
+|  "tweezers", |    0.48                  |
+|  "battery",  |    0.80                  |
+
+Average Precision (AP) is defined as an area under
+the [precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall)
+curve.
+
+## Inputs
+
+Image, name: `images`, shape: `1, 1, 416, 416` in the format `B, C, H, W`, where:
+
+- `B` - batch size
+- `C` - number of channels
+- `H` - image height
+- `W` - image width
+
+Expected color order is `BGR`.
+
+## Outputs
+
+The net outputs blob with shape: `1, 1, 200, 7` in the format `1, 1, N, 7`, where `N` is the number of detected
```

Comment:
output info does not correspond to configuration file (it is ssd DetectionOutput description)

Reply:
and here also no changes, so which output format should be used for your models?

this section should explain how looks raw output of model without postprocessing, if it contains correct info, then probably your ac config wrong. yolo_v3 adapter parse yolo v3 output, which very different from this description

Reply:
hi, here is update.

We are using yolox-nano models with classes-10 or classes-3. 
Can we use yolox adapter for accuracy evaluation? or do we need a custom evaluation script?